core/src/main/java/org/apache/carbondata/core/util/CarbonUtil.java

```diff
@@ -2179,7 +2179,11 @@ static DataType thriftDataTypeToWrapperDataType(
         return DataTypes.FLOAT;
       case BYTE:
         return DataTypes.BYTE;
+      case BINARY:
+        return DataTypes.BINARY;
       default:
+        LOGGER.warn(String.format("It can't match the data type, so use default data type: %s",
```

Comment:
modify log message to "cannot match the data type, using default String datatype", no need of "it"

Reply:
done